pexec/managed_process_test.go

```diff
@@ -22,6 +25,22 @@ import (
 	"go.viam.com/utils/testutils"
 )
 
+// User for subprocess tests.
+// This looks for TEST_SUBPROC_USER var, otherwise uses current user.
+func subprocUser() *user.User {
+	var userInfo *user.User
+	var err error
+	if usernameFromEnv := os.Getenv("TEST_SUBPROC_USER"); len(usernameFromEnv) > 0 {
+		userInfo, err = user.Lookup(usernameFromEnv)
+	} else {
+		userInfo, err = user.Current()
+	}
+	if err != nil {
+		panic(err)
```

Comment:
Can you just return the error and fail the test? Why panic?

Reply:
pythonism that leaked into my go

fixed in ace033ba